src/components/TodoList/TodoList.tsx

```diff
@@ -1,44 +1,135 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './TodoList.scss';
+import classname from 'classnames';
+import { getTodos } from '../../api/api';
 
-export const TodoList: React.FC = () => (
-  <div className="TodoList">
-    <h2>Todos:</h2>
-
-    <div className="TodoList__list-container">
-      <ul className="TodoList__list">
-        <li className="TodoList__item TodoList__item--unchecked">
-          <label>
-            <input type="checkbox" readOnly />
-            <p>delectus aut autem</p>
-          </label>
-
-          <button
-            className="
-              TodoList__user-button
-              TodoList__user-button--selected
-              button
-            "
-            type="button"
-          >
-            User&nbsp;#1
-          </button>
-        </li>
-
-        <li className="TodoList__item TodoList__item--checked">
-          <label>
-            <input type="checkbox" checked readOnly />
-            <p>distinctio vitae autem nihil ut molestias quo</p>
-          </label>
-
-          <button
-            className="TodoList__user-button button"
-            type="button"
-          >
-            User&nbsp;#2
-          </button>
-        </li>
-      </ul>
+type Props = {
+  userId: (todo: Todo) => void,
+};
+
+export const TodoList: React.FC<Props> = ({ userId }) => {
+  const [inputValue, setInputValue] = useState('');
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [optionValue, setOptionValue] = useState('');
+  const [hasLoadingError, setHasLoadingError] = useState(false);
+
+  const filterTodo = todos
+    .filter(todo => todo.title.toLocaleLowerCase()
+      .includes(inputValue.toLocaleLowerCase()));
```

Comment:
Now you transform input value to lower case on each iteration
It's redundant here
Will be better to create separate variable before the filter method where you can store the input value in lower case
And use this variable in filter